CHANGELOG.md

```diff
@@ -5,14 +5,67 @@ All notable changes to this project will be documented in this file.
+* enable fetching manifest with credentials ([#104](https://github.com/joinmarket-webui/joinmarket-webui/issues/104)) ([cea035d](https://github.com/joinmarket-webui/joinmarket-webui/commit/cea035da5c5cde185931cc62da8344f8d07b79c9))
+* extract error message of text/html response ([#84](https://github.com/joinmarket-webui/joinmarket-webui/issues/84)) ([ffa1afa](https://github.com/joinmarket-webui/joinmarket-webui/commit/ffa1afa31aa311cb6efe2838316ca6850fe77713))
+* fix and dry up font features ([#98](https://github.com/joinmarket-webui/joinmarket-webui/issues/98)) ([ac762c5](https://github.com/joinmarket-webui/joinmarket-webui/commit/ac762c520486e33fc493b3de59ad8b02f6aa5332))
+* hidden balance icon alignment and size ([#52](https://github.com/joinmarket-webui/joinmarket-webui/issues/52)) ([8bb8b7b](https://github.com/joinmarket-webui/joinmarket-webui/commit/8bb8b7b98f6f55855ffed4487cd8a0aba1068e06))
+* improve onboarding mobile layout ([#109](https://github.com/joinmarket-webui/joinmarket-webui/issues/109)) ([4159fac](https://github.com/joinmarket-webui/joinmarket-webui/commit/4159fac2303233faa93c57c2e2cec9faa3b8fbd6))
+* initialize local storage with default values ([#33](https://github.com/joinmarket-webui/joinmarket-webui/issues/33)) ([712b289](https://github.com/joinmarket-webui/joinmarket-webui/commit/712b289ddce94a385529e1cc3f4dd2416f65cbc4))
+* Invalid DOM property 'class' in App ([d2173cd](https://github.com/joinmarket-webui/joinmarket-webui/commit/d2173cdb89793b4da4561e3fd374413158dce7ff))
+* navbar padding ([#48](https://github.com/joinmarket-webui/joinmarket-webui/issues/48)) ([1a04910](https://github.com/joinmarket-webui/joinmarket-webui/commit/1a04910f61b0d5b2251065cb754e7f89dae52728))
+* navigate to root on unmapped path ([#89](https://github.com/joinmarket-webui/joinmarket-webui/issues/89)) ([6f0f515](https://github.com/joinmarket-webui/joinmarket-webui/commit/6f0f515e87c177c8e5bf71009782497f779eac83))
+* prevent dom validation warning ([#5](https://github.com/joinmarket-webui/joinmarket-webui/issues/5)) ([88e2ed6](https://github.com/joinmarket-webui/joinmarket-webui/commit/88e2ed66b766662cc3c79f938396c83d2d2a190c))
+* show loading indicator until all data is fetched ([#3](https://github.com/joinmarket-webui/joinmarket-webui/issues/3)) ([37d882f](https://github.com/joinmarket-webui/joinmarket-webui/commit/37d882f9fb61faff2b476f63f92610fefd220473))
+* update sat symbol ([#36](https://github.com/joinmarket-webui/joinmarket-webui/issues/36)) ([765c9d7](https://github.com/joinmarket-webui/joinmarket-webui/commit/765c9d7315f96843cefeddbe51b49682f652ea2a))
+* use sprite instead of plain logo ([215e2fa](https://github.com/joinmarket-webui/joinmarket-webui/commit/215e2fa9a68ad7bef79a80330a45b903d485f160))
```

Comment:
```suggestion
* display info text on send page if service is running ([#86](https://github.com/joinmarket-webui/joinmarket-webui/issues/86)) ([4070d70](https://github.com/joinmarket-webui/joinmarket-webui/commit/4070d7026c0ba7eac7fca02a3c14c65da510f67b))
* fix and dry up font features ([#98](https://github.com/joinmarket-webui/joinmarket-webui/issues/98)) ([ac762c5](https://github.com/joinmarket-webui/joinmarket-webui/commit/ac762c520486e33fc493b3de59ad8b02f6aa5332))
* improve onboarding mobile layout ([#109](https://github.com/joinmarket-webui/joinmarket-webui/issues/109)) ([4159fac](https://github.com/joinmarket-webui/joinmarket-webui/commit/4159fac2303233faa93c57c2e2cec9faa3b8fbd6))
* navigate to root on unmapped path ([#89](https://github.com/joinmarket-webui/joinmarket-webui/issues/89)) ([6f0f515](https://github.com/joinmarket-webui/joinmarket-webui/commit/6f0f515e87c177c8e5bf71009782497f779eac83))
* update sat symbol ([#36](https://github.com/joinmarket-webui/joinmarket-webui/issues/36)) ([765c9d7](https://github.com/joinmarket-webui/joinmarket-webui/commit/765c9d7315f96843cefeddbe51b49682f652ea2a))
```

This would  reduce the changelog a lot. However, it is clearer to state the most notable changes and collect only information valuable to the user in a changelog. I can not evaluate whether someone needs to know that we  "extract error message of text/html response"? I am fine either way. Just wanted to let you know.